src/m365/pa/commands/app/app-get.spec.ts

```diff
@@ -149,7 +149,667 @@ describe(commands.APP_GET, () => {
+  });
+
+  it('retrieves information about the specified app using displayName', async () => {
+    const apps = [
```

Comment:
It's not really part of the scope, but I noticed that this array of apps is used in a lot of tests, which really clutters the test file with hundreds of lines of duplicated code.
Can we declare this list of apps globally over the entire file and reference this variable in the tests?
It would be cool if you could reference this variable in other tests as well, but since it's not part of the scope, it's ok if you use it only in your tests.

Reply:
For now, I've declared the list of apps only inside this test.